### PR TITLE
Chore implement seo best practices

### PR DESCRIPTION
**Pull request by marsolekk**

### CODE REVIEW

File: company_website/templates/common_partials/head.haml [hunk 1]
```diff
@@ -18,6 +19,10 @@
 %meta{:name => "viewport",
         :content => "width=device-width, initial-scale=1, maximum-scale=5, user-scalable=yes"}
 
+{% if request.get_full_path|is_pagination_suffix %}
+    %meta{:name => "robots", :content => "noindex,follow"}
```
Comment (kbeker) [inline company_website/templates/common_partials/head.haml:23]: Missing space between `noindex` and `follow` :)

**Review by kbeker — COMMENTED** [review]
Does it affect anyhow on SEO/performance etc. tests done by google tool?

File: company_website/templates/common_partials/head.haml (hunk 1 above)
Comment (marsolekk) [reply company_website/templates/common_partials/head.haml:23]: Fixed

**Comment by marsolekk** [comment]
> Does it affect anyhow on SEO/performance etc. tests done by google tool?

Not really. Changes have been implemented to follow convention and best practices. There are still some issues pointed by ContentKing (currently tested SEO audit tool) but not urgent.
Google tool like Lighthouse can't catch changes, because is only monitoring presence of tags and not correctness, so this not affect our SEO score in Lighthouse but can have positive impact on our position in google search.

**Review by kbeker — APPROVED** [review]

**Pull request merged**